cfme/tests/infrastructure/test_tenant_quota.py

```diff
@@ -171,3 +186,31 @@ def test_tenant_quota_vm_reconfigure(appliance, provider, setup_provider, set_ro
     setattr(new_config.hw, custom_prov_data['change'], custom_prov_data['value'])
     small_vm.reconfigure(new_config)
     assert small_vm.configuration != original_config
+
+
+@pytest.mark.parametrize(
+    ['parent_quota', 'child_quota', 'flash_text'],
+    [
+        [('cpu', '1'), ('cpu', '2'), 'Cpu'],
+        [('memory', '1'), ('memory', '2'), 'Mem'],
+        [('storage', '1'), ('storage', '2'), 'Storage'],
+        [('vm', '1'), ('vm', '2'), 'Vms'],
+        [('template', '1'), ('template', '2'), 'Templates']
+    ],
+    ids=['cpu', 'memory', 'storage', 'vm', 'template']
+)
+def test_setting_child_quota_more_than_parent(tenants_setup, parent_quota, child_quota,
+                                              flash_text):
+    test_parent, test_child = tenants_setup
+    view = navigate_to(test_parent, 'ManageQuotas', wait_for_view=True)
```

Comment:
**REQUIRED:** Is there a ManageQuota method we can use instead of trying to interact with the forms directly.

Reply:
@psav :No, there is a method "set_quota" which takes kwargs as argument but here we need to pass the kargs with different variable name to set different parameters, so with parameterization it's not possible.